connectors/README.md

```diff
@@ -42,15 +47,21 @@ For example:
   specified, then Google Cloud Storage connector will be updated to 1.7.0 version and BigQuery connector
   will be updated to 0.11.0 version:
   ```
-  gcloud dataproc clusters create <CLUSTER_NAME> \
-      --initialization-actions gs://$MY_BUCKET/connectors/connectors.sh \
+  REGION=<region>
+  CLUSTER_NAME=<cluster_name>
+  gcloud dataproc clusters create ${CLUSTER_NAME} \
+      --region ${REGION} \
+      --initialization-actions gs://goog-dataproc-initialization-actions-${REGION}/connectors/connectors.sh \
       --metadata gcs-connector-version=1.7.0
   ```
 * if Google Cloud Storage connector 1.8.0 version is specified and BigQuery connector version is not
   specified, then only Google Cloud Storage connector will be updated to 1.8.0 version and BigQuery
   connector will be left intact:
```

Comment:
(not sure if this is clearer) intact -> unchanged

Reply:
Done.